src/supporting-data/citations/components/results/__tests__/CitationCard.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { fireEvent, screen } from '@testing-library/react';
+
+import renderWithRouter from '../../../../../shared/__test-helpers__/RenderWithRouter';
+import CitationCard from '../CitationCard';
+
+import citationData from '../../__mocks__/literatureCitationData';
+
+describe('CitationCard tests', () => {
+  it('should render the card component', () => {
+    const { asFragment } = renderWithRouter(
+      <CitationCard data={citationData} />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should allow card selection and navigation', () => {
+    const handleClick = jest.fn();
+    const { history } = renderWithRouter(
+      <CitationCard data={citationData} handleEntrySelection={handleClick} />
+    );
+    fireEvent.click(screen.getByRole('checkbox'));
+    expect(handleClick).toHaveBeenCalled();
+    fireEvent.click(screen.getAllByRole('button')[0]);
+    const {
+      location: { pathname },
+    } = history;
+    expect(pathname).toMatch('/citations/14702039');
```

Comment:
```suggestion
    expect(history.location.pathname).toMatch('/citations/14702039');
```
I think the non-destructured version improved readability